uiautomator-espresso/python/README.md

```diff
@@ -0,0 +1,110 @@
+# Python samples using UIAutomator/ Espresso framework
+
+## I. Setup environment on Mac
+
+#### Homebrew
+
+- [Homebrew](https://brew.sh/) is a package manager for the Mac.
+-  To install Homebrew, open terminal and type the following command:
+
+```bash
+ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+- This will install Homebrew on your Mac. To check the version, type the following command:
+
+```bash
+brew -v
+```
+
+#### Python
+
+- To install Python via Homebrew, type the following command:
+
+```bash
+brew install python
+```
+
+- To check if you have [Python](https://www.python.org/) installed, run this command in your terminal:
+
+```bash
+python --version
+python3 --version
```

Comment:
We should choose python or python3 to run the tests.

Reply:
I want to show that we can run this script with both python & python3.
Consider removing python & keep python3.